docs/manual/figures/value-subtyping.svg

```diff
@@ -1,791 +1,183 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Addressed in d32a700c83489360e6dbeda163d0c297abffb0f5, made the boxes 10px wider.